sda/cmd/finalize/finalize.md

```diff
@@ -23,47 +25,44 @@ export LOG_FORMAT="json"
 
 ### RabbitMQ broker settings
 
-These settings control how finalize connects to the RabbitMQ message broker.
+These settings control how `finalize` connects to the RabbitMQ message broker.
 
 - `BROKER_HOST`: hostname of the RabbitMQ server
-- `BROKER_PORT`: RabbitMQ broker port (commonly `5671` with TLS and `5672` without)
-- `BROKER_QUEUE`: message queue to read messages from (commonly `accessionIDs`)
-- `BROKER_ROUTINGKEY`: message queue to write success messages to (commonly `backup`)
+- `BROKER_PORT`: RabbitMQ broker port (commonly: `5671` with TLS and `5672` without)
+- `BROKER_QUEUE`: message queue to read messages from (commonly: `accessionIDs`)
+- `BROKER_ROUTINGKEY`: message queue to write success messages to (commonly: if `ARCHIVE_` and `BACKUP_` storage backend are set queue should be `completed` otherwise `backup`)
```

Comment:
```suggestion
- `BROKER_ROUTINGKEY`: Routing key for publishing messages (commonly: `completed`)
```

Messages is always sent to `completed` now since `finalize` performs archive backup if configured to do so.